README.md

```diff
@@ -135,6 +136,7 @@ See also https://github.com/timotheecour/D_vs_nim/issues/11
 | increment | i++ | i+=1 or inc(i) |
 | concatenation | ~ | & |
 | empty statement | {} | discard |
+| null pointer | `null` | `nil` |
 | **syntax:parsing** |
```

Comment:
> I researched these features because I was wondering if either language had a solution like Rust’s Option. Neither language does.

https://dlang.org/phobos/std_typecons.html#.Nullable
https://nim-lang.org/docs/options.html

so maybe add a short entry under `library` since (as you've experienced) it's hard to find (especially given name differences) ; just showing module name is enough IMO, interested users can dig further once they know where to look

Reply:
I added a row “option/maybe type” within the “library” section of the “map of corresponding features” table.